frontend/packages/dev-console/console-extensions.json

```diff
@@ -650,5 +664,13 @@
     "flags": {
       "required": ["ALLOW_SERVICE_BINDING"]
     }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": true,
+      "path": ["/project-access/ns/:ns"],
```

Comment:
Can we also add `/project-access/all-namespaces` path. and handle the `all-namespaces` use case same as we do for other dev console pages?

Currently `/project-access/all-namespaces` shows 404 page.